contrib/backporting/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:20.04
+
+LABEL maintainer="maintainer@cilium.io"
+
+RUN apt-get update && DEBIAN_FRONTEND="noninteractive" apt-get -y install tzdata
+RUN apt-get install -y \
+  git \
+  jq \
+  python3 \
+  python3-pip \
+  curl \
+  vim
+RUN pip3 install --user PyGithub
+RUN mkdir -p /hub && \
+    cd /hub \
+    && curl -L -o hub.tgz https://github.com/github/hub/releases/download/v2.14.0/hub-linux-amd64-2.14.0.tgz \
+    && tar xfz hub.tgz \
+    && $(tar tfz hub.tgz | head -n1 | cut -f1 -d"/")/install \
```

Comment:
We should do a SHA checksum match here as well

Reply:
Sorry, I don't quite follow. Can you elaborate more?

Reply:
Hm, I assumed that GH would provide the SHA sum file to check against when we download a release, but they don't provide any :confused:.

https://github.com/github/hub/releases